src/Missings.jl

```diff
@@ -208,6 +208,104 @@ missing
 """
 passmissing(f) = PassMissing{Core.Typeof(f)}(f)
 
+struct SpreadMissings{F} <: Function
+    f::F
+end
+
+function (f::SpreadMissings{F})(x) where {F}
```

Comment:
Do you really need a specific method for a single `x`?

Reply:
no, its unnecessary.